src/WebGPU/pointer.ts

```diff
@@ -179,6 +177,27 @@ export default function initMouseController(
         event.preventDefault()
         Logic.commitChanges()
         break
+      case '=':
+      case '+':
+        // Zoom in with Ctrl/Cmd + Plus
+        if ((!event.ctrlKey && !event.metaKey)) break
+        event.preventDefault()
+        {
```

Comment:
in this case if we need a block anyway, lets change the condition above, instead of ` if ((!event.ctrlKey && !event.metaKey)) break` let's do ` if ((event.ctrlKey || event.metaKey)) {` and same for minuts